libc/test/UnitTest/CMakeLists.txt

```diff
@@ -116,30 +150,22 @@ add_unittest_framework_library(
   HDRS
     FEnvSafeTest.h
     FPMatcher.h
+    FPTest.h
     RoundingModeUtils.h
   DEPENDS
     LibcTest
+    ErrnoSafeTest
```

Comment:
We usually add `.ErrnoSetterMatcher` as relative paths. Is it better to unify the style?

cc @lntue

Reply:
I'm down to make the change. Should we change the other names in this file too? lmk

Reply:
ping @SchrodingerZhu

Reply:
Ah, sorry for missing the reply. I am fine with unifying all of them at once.